packages/core/core/src/plugin-utils.ts

```diff
@@ -51,7 +51,7 @@ export class Plugin<PluginConfig> {
 export interface StorageHandler {
   logger: Logger;
   deletePackage(fileName: string): Promise<void>;
-  removePackage(): Promise<void>;
+  removePackage(packageName: string): Promise<void>;
```

Comment:
I think this method is fine, the remove package is only meant to remove the folder, usually only when there are no package left.

<img width="664" alt="Screenshot 2025-03-08 at 13 29 27" src="https://github.com/user-attachments/assets/3514dfad-2bde-4ca3-8571-79ec6a82d887" />

Reply:
That's true for file-based storage. 

I'm implementing a SQL/Postgres/Drizzle plugin (should work for multiple Verdaccio instances, scalable, no locking issues). The storage handler is a singleton since I don't want instances for each package. Without the package parameter, removePackage doesn't know what to remove.

Reply:
ok make sense, let's go with it.